haiku-apps/bdhcalc/bdhcalc-1.1.recipe

```diff
@@ -27,6 +27,10 @@ BUILD_PREREQUIRES="
 	cmd:g++
 	"
 
+if [ "$effectiveTargetArchitecture" == "x86_64" ] ; then
```

Comment:
* If it's a gcc2 vs gcc5 test, then you should use:
```
if [ "$effectiveTargetArchitecture" != x86_gcc2 ] ; then
```
(This is because an (unsupported) Haiku/x86 exists and is 32 bits with gcc5.)

* If it's a 32 bit vs 64 bis issue, then I would recommend using `=` instead of `==`
```
if [ "$effectiveTargetArchitecture" = x86_64 ] ; then
```
(Using `==` is a bashism. `=` works on `sh` and `bash`, while `==` only works on `bash`)

* BTW, you can keep the quotes if you wish, but, as you certainly know, they are not required in string constants (as long as they don't have unprotected white spaces.)